<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAWConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>391</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>1201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>601</width>
      <height>581</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>599</width>
       <height>579</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>630</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>630</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomInButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxHalfsizeinterpolation">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>160</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Half size interpolation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFourColorInterpolation">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Four Color Interpolation</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxStrechOrRotate">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't strech or rotate raw pixels</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDocumentMode">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>230</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Document Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAverageWB">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>250</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Average white balans</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>280</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Highlights</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDontAutoBrighten">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string> Don't automatically brighten the image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCameraWb">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>110</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Use camera white balans</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxNoColorNoInterpolation">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>310</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>No color, no interpolation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxHighlights">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>270</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Remove shadows</string>
    </property>
    <item>
     <property name="text">
      <string>Remove shadows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Don't remove shadows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Safe remove shadows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recovery shadows 5</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBoxChangeDateToCameraTime">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>330</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Change file dates to camer timestamp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRotateImage">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>350</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Don't rotate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflection vertical axis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflection horizontal axis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180 degree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflection vertical axis and rotate 90 degree to right</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90 degree to left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90 degree to right</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflection vertical axis and rotate 90 to left</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxQuality">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>410</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Poor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Standart</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Good</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ultra</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>390</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolation Quality</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>440</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output colorspace</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>460</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Raw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sRGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Adobe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wide</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ProPhoto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XYZ</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>490</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBrigtness">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>510</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>540</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set threshhold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>560</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
